### PR TITLE
Add sql-migration extension V0.1.8 to stable extension gallery

### PR DESCRIPTION
**Pull request by rchlkm**
Build: https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=116260&view=artifacts&pathAsName=false&type=publishedArtifacts

Inline Migration service will be in public preview on 9/1/21. We would like to merge this PR and publish the changes on the morning of 9/1.

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -4077,6 +4077,63 @@
...
+					"shortDescription": "This extension lets you migrate your SQL Server instance to Azure SQL.",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.8",
+							"lastUpdated": "08/18/2021",
```
Comment (alanrenmsft) [inline extensionsGallery.json:4093]: fix the date

Comment (rchlkm) [reply extensionsGallery.json:4093]: this is the same extension build as the one released on the insider's on 8/18. should it still be updated to today's or 9/1 date?

**Comment by alanrenmsft** [comment]
for stable gallery, the build should be for the latest release branch, or at least the main branch but the one you have is pointing to a private branch.

**Comment by alanrenmsft** [comment]
I can take care of this on Sep 1st.

File: extensionsGallery.json (hunk 1 above)
Comment (alanrenmsft) [reply extensionsGallery.json:4093]: not a huge deal, i guess it is fine

**Comment by Charles-Gagnon** [comment]
@alanrenmsft Me and @aasimkhan30 had a long discussion about this - for this extension they're going to be releasing out of other branches as needed for their scenarios.

**Comment by aasimkhan30** [comment]
Unless, I am missing something it seems only the gallery needs to be updated. There is already 0.1.8 vsix available in the blob storage.

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
Gallery updated!

**Comment by Charles-Gagnon** [comment]
@rchlkm FYI you may want to consider adding this to the list of recommended extensions as well : https://github.com/microsoft/azuredatastudio/blob/main/product.json#L42

**Pull request merged**